Sources/RoutingKit/TrieRouter.swift

```diff
@@ -53,7 +66,7 @@ public final class TrieRouter<Output>: Router, CustomStringConvertible {
 
         // if this node already has output, we are overriding a route
         if current.output != nil {
-            print("[Routing] Warning: Overriding route output at: \(path.string)")
+            self.logger.warning("[Routing] Overriding route output at: \(path.string)")
```

Comment:
I think this should probably be info/debug? Warning seems too high and goes against the logging guidelines

Reply:
Yeah, agreed.
```suggestion
            self.logger.info("[Routing] Overriding route output at: \(path.string)")
```